src/main/java/org/breedinginsight/brapi/v2/BrAPIProgramsController.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.breedinginsight.brapi.v2;
+
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.annotation.*;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+import lombok.extern.slf4j.Slf4j;
+import org.brapi.v2.model.BrAPIIndexPagination;
+import org.brapi.v2.model.BrAPIMetadata;
+import org.brapi.v2.model.core.BrAPIProgram;
+import org.brapi.v2.model.core.response.BrAPIProgramListResponse;
+import org.brapi.v2.model.core.response.BrAPIProgramListResponseResult;
+import org.brapi.v2.model.core.response.BrAPIProgramSingleResponse;
+import org.breedinginsight.api.auth.ProgramSecured;
+import org.breedinginsight.api.auth.ProgramSecuredRoleGroup;
+import org.breedinginsight.api.auth.SecurityService;
+import org.breedinginsight.brapi.v1.controller.BrapiVersion;
+import org.breedinginsight.model.Program;
+import org.breedinginsight.services.ProgramService;
+
+import javax.annotation.Nullable;
+import javax.inject.Inject;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Controller("/${micronaut.bi.api.version}")
```

Comment:
Because this controller defines both root and program-scoped endpoints, I think the `"/${micronaut.bi.api.version}"` string should be prepended to the program-scoped endpoints only. As is, I can see `/serverinfo`, `/programs` and `/programs/{programDbId}` endpoints listed in the response to http://localhost:8081/brapi/v2/serverinfo, but the root programs endpoints are actually under `http://localhost:8081/v1/brapi/v2/programs[/{programDbId}] (note the `v1`) which is unexpected.

Reply:
I'll make the change.